CondTools/SiPhase2Tracker/BuildFile.xml

```diff
@@ -0,0 +1,10 @@
+<use name="FWCore/Framework"/>
+<use name="FWCore/ParameterSet"/>
+<use name="FWCore/ServiceRegistry"/>
+<use name="CondCore/DBOutputService"/>
+<use name="CondFormats/Common"/>
+<use name="CondFormats/SiPhase2TrackerObjects"/>
+<use name="CondFormats/DataRecord"/>
+<use name="rootrflx"/>
+<!--<use name="boost"/> -->
```

Comment:
can be the commented out code be cleaned up?

Reply:
Yes